build.xml

```diff
@@ -55,8 +55,8 @@ xmlns:cs="antlib:com.puppycrawl.tools.checkstyle.ant">
     <property name="jackson-mapper-asl.version" value="1.9.11"/>
     <property name="dependency-check-ant.version" value="2.1.0"/>
 
-    <property name="commons-io.version" value="2.4"/>
-    <property name="kerby.version" value="1.0.0-RC2"/>
+    <property name="commons-io.version" value="2.5"/>
```

Comment:
2.6 is available - why not update to that?

Reply:
It is the only requirement for Kerby. Will update to 2.6